qubesbuilder/plugins/__init__.py

```diff
@@ -324,6 +324,16 @@ def default_copy_in(self, plugins_dir: Path, sources_dir: Path):
                         sources_dir,
                     )
                 )
+            if dependency.builder_object == "job":
+                artifact_path = get_artifact_path(
+                    self.config, dependency.reference
+                )
+                for artifact in artifact_path.parent.iterdir():
+                    if artifact == artifact_path:
+                        continue
+                    copy_in.append(
+                        (artifact, self.executor.get_dependencies_dir())
+                    )
```

Comment:
should they all be copied together, or maybe better in separate subdirs? what if file names conflict (for example you declare QWT use vm-win10 and vm-win11 if that would exist)?